development/playbooks/lock/lock.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Lock production collections
```

Comment:
Right now, this generates a `requirements.yml`-style file, based on the data `ansible-galaxy collection list` gives us.
This means:
- the versions are what is in the respective `galaxy.yml` (a pure string, no git reference or anything, if installed from git)
- an actual `ansible-galaxy install` run needs to have happened before this.

Thus, this is quite different to the way we utilize librarian in `foreman-installer`. I don't think that's bad as such, just probably unexpected by seasoned librarian users.